packages/charting/src/components/StackedBarChart/StackedBarChart.base.tsx

```diff
@@ -53,10 +53,31 @@ export class StackedBarChartBase extends React.Component<IStackedBarChartProps,
 
   public render(): JSX.Element {
     this._adjustProps();
-    const { data, hideNumberDisplay, hideLegend, theme, styles, barBackgroundColor, href, ignoreFixStyle, hideDenominator } = this.props;
+    const {
+      data,
+      benchmarkData,
+      targetData,
+      hideNumberDisplay,
+      hideLegend,
+      theme,
+      styles,
+      barBackgroundColor,
+      href,
+      ignoreFixStyle,
+      hideDenominator
+    } = this.props;
     const { palette } = theme!;
     const barHeight = ignoreFixStyle || data!.chartData!.length > 2 ? this.props.barHeight : 8;
-    const bars = this._createBarsAndLegends(data!, barHeight!, palette, barBackgroundColor, href);
+    if (benchmarkData) {
+      benchmarkData.color = benchmarkData.color || '#A19F9D';
```

Comment:
Looks like there's lots of theme color examples in here already, though it's using the palette rather than semantic colors.

Just add semanticColors to the destructure on line 69 and you'll have lots available.

Reply:
We can use the theme palette color to replace the hard code color. It is not necessary to add semanticColors to the destructure.

---
In reply to: [324845982](https://github.com/OfficeDev/office-ui-fabric-react/pull/10455#discussion_r324845982) [](ancestors = 324845982)

Reply:
palette.neutralTertiary is equal to gray90
palette.neutralSecondary is equal to gray130.

Can't find gray140 in theme.palette and theme.semanticColors. So use palette.neutralSecondary as a workaround.

---
In reply to: [324756894](https://github.com/OfficeDev/office-ui-fabric-react/pull/10455#discussion_r324756894) [](ancestors = 324756894)

Reply:
@mltejera can help with the theme related questions. gray90 and gray140 don't exist in the semantic slots (the names) so we'll need to map it to values that make sense.

Reply:
Agreed. 

SemanticColors have a ton of benefits over the base theme palette. The biggest one being that they are far more likely to work well together if/when the base palette changes. I strongly encourage you to use them going forward. They end up being a lot less work to implement once you're familiar with them.

https://github.com/OfficeDev/office-ui-fabric-react/wiki/Theming

https://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/theming-designer/index.html

Reply:
Does this mean that the palette color has been deprecated, and now only the semantic color should be used in the code? If so, Can you give me some guidance on how to map the colors on the design to the right SemanticColors? I feel very confused about it.
For example. 
![image](https://user-images.githubusercontent.com/12230173/65104041-adab8680-da02-11e9-87de-7ce1cc446cc6.png)
Benchmark: gray90
Target: gray140
How to map this color to values that make sense.

![image](https://user-images.githubusercontent.com/12230173/65103957-691feb00-da02-11e9-8772-052cd0fc1569.png)

Reply:
Agreed, if you're given the different numbered grays as spec, it adds a layer of complexity to finding the right semantic colors.

The color palette hasn't been deprecated per say, but we should trace it through to the right semantic slot.

One place the semantic color system breaks down is in graphical decorations - the bars in charts being one of them. So for these, I'd be okay using the closest base palette color.

Reply:
That's okay. Great to know that I don't need to change it. 
​​​​​​​
​​​​​​​